test/container.test.ts

```diff
@@ -165,12 +166,16 @@ test('should flush the beacon queue on initialization', async () => {
     expect(server).toReceiveMessage(expect.objectContaining({payload: payload}));
 });
 
-test('should configure a fixed CID assigner if a CID is specified', async () => {
+test.each([
+    [true],
+    [false],
+])('should configure a fixed CID assigner if a CID is specified', async (test: boolean) => {
```

Comment:
This one sounds confusing to me, given that there's a similar test above. What do you think about `should configure a fixed CID assigned with a specific CID if specified`?